protocols/delegate-manager/test/DelegateManager-unit.js

```diff
@@ -0,0 +1,391 @@
+const Delegate = artifacts.require('Delegate')
+const Indexer = artifacts.require('Indexer')
+const DelegateManager = artifacts.require('DelegateManager')
+const DelegateFactory = artifacts.require('DelegateFactory')
+const MockContract = artifacts.require('MockContract')
+const ERC20 = artifacts.require('ERC20')
+const { equal, passes, reverted } = require('@airswap/test-utils').assert
+const { takeSnapshot, revertToSnapShot } = require('@airswap/test-utils').time
+const { EMPTY_ADDRESS } = require('@airswap/order-utils').constants
+const { padAddressToLocator } = require('@airswap/test-utils').padding
+
+contract('DelegateManager Unit Tests', async accounts => {
+  let owner = accounts[0]
+  let tradeWallet = accounts[1]
+  let mockDelegate
+  let mockIndexer
+  let delegateManager
+  let mockFactory
+  let mockSwap
+  let mockWETH
+  let mockDAI
+  let mockStakeToken
+  let mockStakeToken_allowance
+  let mockStakeToken_transferFrom
+  let mockStakeToken_transfer
+  let mockStakeToken_approve
+  let mockIndexer_getScore
+  let snapshotId
+
+  beforeEach(async () => {
+    let snapShot = await takeSnapshot()
+    snapshotId = snapShot['result']
+  })
+
+  afterEach(async () => {
+    await revertToSnapShot(snapshotId)
+  })
+
+  async function setupMockTokens() {
+    mockWETH = await MockContract.new()
```

Comment:
why not just make these be `accounts[i]` addresses? There's no behaviour that needs mocking as they are just addresses to the delegate and indexer contracts

Reply:
will do.